### PR TITLE
Hosted Deep Link Data in banner and deepview.

### PR DESCRIPTION
**Pull request by TheMarshal99**
Updated the banner() and deepview() calls.
* Will now pull in any hosted deep link data into the data block.
* Items in the data block will override hosted deep link data.

@jsaleigh

### CODE REVIEW

File: src/1_utils.js [hunk 1]
```diff
@@ -439,4 +439,21 @@ utils.scrapeOpenGraphContent = function(property, content) {
...
+/**
+ * Search for hosted deep link data on the page, as outlined here https://dev.branch.io/getting-started/hosted-deep-link-data/guide/#adding-metatags-to-your-site
+ */
+utils.scrapeHostedDeepLinkData = function() {
+	var params = {};
+
+	var metas = document.getElementsByTagName("meta");
+
+	for (var i = 0; i < metas.length; i++) {
+		if (metas[i].getAttribute("name").startsWith("branch:deeplink:")) {
```
Comment (jsaleigh) [inline src/1_utils.js]: If there is no 'name' attribute, `getAttribute` will return null, then `startsWith` will crash

Comment (jsaleigh) [inline src/1_utils.js]: `startsWith` cannot be used:

http://kangax.github.io/compat-table/es6/#test-String.prototype.startsWith

**Comment by TheMarshal99** [comment]
@jsaleigh ^^

**Comment by yntema** [comment]
👍  looks good!

**Pull request merged**